### PR TITLE
Prepared for eXist 3.0.RC2

### PR DESCRIPTION
**Pull request by adamretter**
**NOTE** Not to be merged until `eXist-3.0.RC2` is released

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -2,6 +2,7 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+
     <groupId>org.exist-db</groupId>
     <artifactId>XMLCalabash</artifactId>
     <version>0.1.7-SNAPSHOT</version>
```
Comment (shabanovd) [inline pom.xml]: this should be increased too

Comment (adamretter) [inline pom.xml]: Yes but only when this is released not when eXist is released.

Comment (shabanovd) [inline pom.xml]: better to change on 1st change after release

**Comment by adamretter** [comment]
Closes https://github.com/eXist-db/eXist-XMLCalabash/issues/3

**Pull request merged**